data/coalition/resistance intro.txt

```diff
@@ -1661,27 +1806,37 @@ mission "Resistance: Quarg Interview"
 			`	For a few seconds, they simply exchange looks at your question, until Lobmab speaks up. "Transmissions and messages from some source, Bebliss has received for centuries now. Told us they have of Heliarch activity, fleet movements, training drills, propaganda plans. Always exactly right on their predictions, the messages were. Also pointed us to equipment caches - caches with outfits previously found nowhere in the Coalition - they have. Human weaponry, blueprints, factory parts..."`
 			`	"Aware of the identity of whoever does it, we are not," Elirom says. "An arbiter I was, and though a great deal I knew, still kept from me, some critical information was. Possible, it is, that from one of the consuls, the information comes. Though, difficult to steal a jump drive to bring in the weapons, it would be, even for a consul." He pauses, a dead stare in his eyes as he bites lightly on a finger. "Word of... experiments, we have also received from the messages. Some manner of procedure to take over bodies... stop whatever it is they're doing in the rings, we must."`
 				goto questions
 			label pay
+			action
+				log "Factions" "Lunarium Redoubt" "Almost all of the funding for the Lunarium comes from the great fortune of House Bebliss. Houses Bebliss and Idriss also act as intermediaries, taking out loans, on behalf of the Lunarium, with House Plumtab, who while interested in removing the Heliarchs from power, are not interested in interacting with the Lunarium directly."
 			`	The rest of the table look to Lobmab at the same time, prompting her explanation. "Amassed great fortune over our history, House Bebliss has. As part of our gracious help to the Lunarium, providing nearly everything in financial terms we have been."`
 			`	She stops, with the others still looking at her, as if expecting something more. She remains silent, so Chiree looks back at you, continuing, "Also taken some loans from House Plumtab in our stead, Houses Bebliss and Idriss have. Not terribly passionate for our cause, Plumtab's bankers are, but share our interest in booting the Heliarchs, they do, even if meet or talk with us, they never do."`
 				goto questions
 			label task
+			action
+				log "Agreed to transport Oobat to meet the Quarg living in human space. Since no Quarg ship has been seen in the Coalition in millenia, she believes they have moved on from the war. And that, if the populace can be shown that the Quarg are not interested in invading, the rule of the Heliarchs, who rely on fear of the Quarg to maintain their power, will be weakened. To this end, she wishes to meet a Quarg and record and interview for broadcast in the Coalition."
```

Comment:
Not really sure if I'm just not understanding what's being said or if there really is an issue.

```suggestion
				log "Agreed to transport Oobat to meet the Quarg living in human space. Since no Quarg ship has been seen in the Coalition in millenia, she believes they have moved on from the war. And that, if the populace can be shown that the Quarg are not interested in invading, the rule of the Heliarchs, who rely on fear of the Quarg to maintain their power, will be weakened. To this end, she wishes to meet a Quarg and record an interview to broadcast in the Coalition."
```